src/ResourceManager/RecoveryServices/Commands.RecoveryServices/packages.config

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.4.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.RecoveryServices" version="1.0.0-preview" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
```

Comment:
everything else is using 1.0.9.  This should be ok, but you should have a very good reason to use a different version